site/docs/resources/about.md

```diff
@@ -0,0 +1,53 @@
+# About grammY
+
+## What is grammY?
+
+grammY is a piece of software that you can use when you want to program your own chat bot for [the Telegram messenger](https://telegram.org).
+When you make bots, you will notice that some parts of this process are tedious and always the same.
+grammY does the heavy lifting for you and makes it super simple to create a bot.
+
+## When was grammY created?
+
+The first publish of grammY code was in late March, 2021.
+It reached the first stable version a few weeks later.
+
+## Who develops grammY?
+
+grammY is developed by a team of volunteers who work on the core library, the docs, and the remaining ecosystem in their free time.
+You can find a list of all contributors [in the README of our repository](https://github.com/grammyjs/grammY#contributors-).
+
+Feel free to take contact [in the group chat](https://t.me/grammyjs)!
+If you speak Russian, you can also join us [here](https://t.me/grammyjs_ru)!
+
+## How is grammY developed?
+
+grammY is completely free and open-source software.
+Its code is [available on GitHub](https://github.com/grammyjs/grammY).
+
+We welcome all contributions.
+All code changes are reviewed by several pairs of eyes, optimized iteratively and tested extensively, often using production code of various projects and people.
+
+## What programming language does grammY use?
+
+grammY is written from the ground up in TypeScript—a superset of JavaScript.
+Therefore, it runs on Node.js.
+
+However, grammY can also run on Deno, which positions itself as the successor of Node.js.
+(Technically, you can even run grammY on modern browsers, though this will rarely be useful.)
+
+## How does grammY compare to its competitors?
+
+If you're coming from a different programming language or framework, you can check out our [detailed comparison between frameworks](./comparison.md).
```

Comment:
```suggestion
If you’re coming from a different programming language or framework, you can check out our [detailed comparison between frameworks](./comparison.md).
```

Reply:
Has this been performed consistently everywhere yet? I think we wanted to have a single PQ that fixes it everywhere at some point

Reply:
Right, it might not have been related to this very much. But you created a new page, and we want it to be first published with the right quotes. :P